cmd/nerdctl/run.go

```diff
@@ -171,6 +171,7 @@ func setCreateFlags(cmd *cobra.Command) {
 	cmd.Flags().StringSlice("cap-drop", []string{}, "Drop Linux capabilities")
 	cmd.RegisterFlagCompletionFunc("cap-drop", capShellComplete)
 	cmd.Flags().Bool("privileged", false, "Give extended privileges to this container")
+	cmd.Flags().Bool("host-process", false, "Allow host access for Windows Containers")
```

Comment:
Should be added only when GOOS==windows

Reply:
since docker does have `--isolation` on Linux should we keep it? I left it for now.